.github/workflows/shared-release.yml

```diff
@@ -100,9 +105,12 @@ jobs:
             EXCLUDE_ARG=$FIRST$SECOND
           fi
           
-          echo IS_DEV_RELEASE=$IS_DEV_RELEASE
-          echo IS_DEV_RELEASE=$IS_DEV_RELEASE >> $GITHUB_ENV
-          echo "::set-output name=is-dev-release::$IS_DEV_RELEASE"
+          echo IS_PRE_RELEASE=$IS_PRE_RELEASE
```

Comment:
At the end of this workflow the "Create GitHub Release" step still uses the `IS_DEV_RELEASE` environment variable, I assume that needs to be changed to `IS_PRE_RELEASE`?

```yml
      - name: Create GitHub release
        if: ${{env.IS_RELEASE && inputs.create-github-release}}
        uses: softprops/action-gh-release@1e07f4398721186383de40550babbdf2b84acfc5
        with:
          body_path: RELEASE_CHANGELOG.md
          prerelease: ${{env.IS_DEV_RELEASE}} <<<<<< HERE
          tag_name: ${{env.CURRENT_VERSION}}
```

Reply:
That could have been awkward, good catch!